packages/rest/src/router/routing-table.ts

```diff
@@ -126,13 +147,16 @@ export class RoutingTable {
         describeOperationParameters(route.spec),
       );
     }
-    this._routes.push(route);
+
+    validatePath(route.path);
+    // Add the route to the trie
```

Comment:
I find this comment misleading, we don't use trie by default. Let's remove it completely.